### DISCUSSION TITLE
After running mod from IDE, playing in vanilla always crashes...

### DISCUSSION THREAD

**Discussion by SirEnder125** [FabricMC/fabric-api#2162] [FAQ] [answered]
Hey, I've had intellij idea running for a while, and I've run my mod multiple times from it. After closing, when I was finished modding, I went to play vanilla. For some reason, the game crashes after running well for a few minutes, and this happens each time... The game is still running, but the rendering pauses... I know it's still live, because I can press ESC and make the cursor show up on the screen, and I can hear myself walking... I hope these two things are related, because every time I open intellij for a long time, minecraft behaves this way.

Any ideas?

**Comment by Technici4n** [comment, marked as answer]
Please provide the crash log?

**Reply by SirEnder125** [reply]
[latest.log](https://github.com/FabricMC/fabric/files/8564878/latest.log)
Didn't actually crash. It was more like a freeze... so I closed the window.

**Reply by Technici4n** [reply]
This is a Forge log, I don't see what this has to do with Fabric.

**Reply by SirEnder125** [reply]
Oh, right. I guess the problem is with IntelliJ iteself, causing my game to freeze, after being run.
